packages/debugger/src/locator/DOMLocator/types.ts

```diff
@@ -0,0 +1,18 @@
+import type {KbdKey} from '@solid-primitives/keyboard'
+import type {TargetIDE, TargetURLFunction} from './find-components'
+
+export type {LocationAttr, LocatorComponent, TargetIDE, TargetURLFunction} from './find-components'
+
+export type LocatorOptions = {
```

Comment:
I could not make LocatorOptions DOM-agnostic without changing the api. targetIDE is fine, but key assumes the browser's implementation of keyboard keys